### PR TITLE
Add WP and o2

### PR DESCRIPTION
**Pull request by mimi89999**

### CODE REVIEW

File: entries/w/wp.pl.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "WP": {
+    "domain": "wp.pl",
+    "url": "https://www.wp.pl/",
+    "tfa": [
+      "totp",
+      "custom-software"
```
Comment (Carlgo11) [inline entries/w/wp.pl.json:5]: Could you please specify the custom software?

File: entries/w/wp.pl.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "WP": {
+    "domain": "wp.pl",
+    "url": "https://www.wp.pl/",
+    "tfa": [
+      "totp",
```
Comment (Carlgo11) [inline entries/w/wp.pl.json]: Can you confirm that the TOTP option works with third party apps (such as Google Authentication, Authy, Duo, 1Password)?

File: entries/w/wp.pl.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "WP": {
+    "domain": "wp.pl",
+    "url": "https://www.wp.pl/",
```
Comment (Carlgo11) [inline entries/w/wp.pl.json]: As `https://wp.pl/` already redirects to `https://www.wp.pl/` this line is unnecessary.
```suggestion
```

File: entries/w/wp.pl.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+  "WP": {
```
Comment (Carlgo11) [inline entries/w/wp.pl.json]: The full name seems to be Wirtualna Polska.
```suggestion
  "Wirtualna Polska ": {
```

File: entries/w/wp.pl.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+    "domain": "wp.pl",
+    "url": "https://www.wp.pl/",
+    "tfa": [
+      "totp",
+      "custom-software"
+    ],
+    "documentation": "https://pomoc.wp.pl/1login/jak-wlaczyc-i-wylaczyc-weryfikacje-dwuetapowa",
+    "keywords": [
+      "email"
+    ]
```
Comment (Carlgo11) [inline entries/w/wp.pl.json:16]:
```suggestion
    ],
    "regions": [
      "pl"
    ]
```

File: img/o/o2.pl.svg [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+    <path
+       d="m 65.372,39.76 a 17.381,17.381 0 0 1 -25.58,0 17.35,17.35 0 0 1 0,-25.56 17.381,17.381 0 0 1 25.58,0 17.349,17.349 0 0 1 0,25.56"
+       fill="#d423b5"
+       id="path15" />
+    <path
+       d="m 65.372,34.79 c -4.946,4.763 -12.82,4.763 -17.764,0 -4.767,-4.941 -4.767,-12.809 0,-17.75 4.945,-4.763 12.818,-4.763 17.764,0 4.767,4.941 4.767,12.809 0,17.75"
+       fill="#ff00a2"
+       id="path17" />
+  </g>
+</svg>
```
Comment (Carlgo11) [inline img/o/o2.pl.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 32 32"><mask id="a" width="72" height="71" x="0" y="0" maskUnits="userSpaceOnUse"><path fill="#fff" fill-rule="evenodd" d="M57.53 0c-6.91 0-12.62 5.2-13.42 11.89A30.83 30.83 0 0031 8.99 31.05 31.05 0 000 39.98C0 57.1 13.91 71 31.02 71a30.84 30.84 0 0015.59-4.2l-4.74-8.14A21.61 21.61 0 019.43 40c0-11.91 9.68-21.6 21.59-21.6a21.61 21.61 0 0121.1 26.16 2.55 2.55 0 01-2.48 2.04 2.54 2.54 0 01-2.54-2.54l-.05-4.1c-.02-8.82-7.2-16-16.03-16-8.84 0-16.04 7.2-16.04 16.03 0 8.84 7.2 16.03 16.04 16.03 3.7 0 7.27-1.33 10.08-3.59a11.93 11.93 0 008.54 3.6 12 12 0 0011.71-9.54c.45-2.12.68-4.3.68-6.5a30.83 30.83 0 00-2.88-13.05A13.52 13.52 0 0057.53 0zM24.16 40.12a6.75 6.75 0 016.75-6.75 6.75 6.75 0 016.75 6.75 6.75 6.75 0 01-6.75 6.74 6.75 6.75 0 01-6.75-6.74zM58.2 12.09c.32-.29.64-.61.97-.97.32-.37.48-.79.48-1.27 0-.55-.2-.97-.6-1.28a2.23 2.23 0 00-1.4-.47c-.65 0-1.15.2-1.51.59-.36.4-.56.88-.61 1.45l-3.45-.25c.04-.83.22-1.55.53-2.16a4.56 4.56 0 011.22-1.54 5.16 5.16 0 011.78-.91 7.72 7.72 0 012.22-.31 6.82 6.82 0 012.07.3 4.85 4.85 0 011.68.91c.48.4.85.9 1.12 1.51.27.6.4 1.31.4 2.11 0 .51-.05.98-.16 1.4a4.68 4.68 0 01-.45 1.13c-.19.34-.42.65-.68.94-.26.29-.54.57-.85.84l-4.95 4.2h7.23v2.99H51.87v-3.6z" clip-rule="evenodd"/></mask><g fill-rule="evenodd" clip-rule="evenodd" mask="url(#a)" transform="matrix(.45035 0 0 .4507 0 0)"><path fill="#fff" d="M57.53 0c-6.91 0-12.62 5.2-13.42 11.89A30.83 30.83 0 0031 8.99 31.05 31.05 0 000 39.98C0 57.1 13.91 71 31.02 71a30.84 30.84 0 0015.59-4.2l-4.74-8.14A21.61 21.61 0 019.43 40c0-11.91 9.68-21.6 21.59-21.6a21.61 21.61 0 0121.1 26.16 2.55 2.55 0 01-2.48 2.04 2.54 2.54 0 01-2.54-2.54l-.05-4.1c-.02-8.82-7.2-16-16.03-16-8.84 0-16.04 7.2-16.04 16.03 0 8.84 7.2 16.03 16.04 16.03 3.7 0 7.27-1.33 10.08-3.59a11.93 11.93 0 008.54 3.6 12 12 0 0011.71-9.54c.45-2.12.68-4.3.68-6.5a30.83 30.83 0 00-2.88-13.05A13.52 13.52 0 0057.53 0zM24.16 40.12a6.75 6.75 0 016.75-6.75 6.75 6.75 0 016.75 6.75 6.75 6.75 0 01-6.75 6.74 6.75 6.75 0 01-6.75-6.74zM58.2 12.09c.32-.29.64-.61.97-.97.32-.37.48-.79.48-1.27 0-.55-.2-.97-.6-1.28a2.23 2.23 0 00-1.4-.47c-.65 0-1.15.2-1.51.59-.36.4-.56.88-.61 1.45l-3.45-.25c.04-.83.22-1.55.53-2.16a4.56 4.56 0 011.22-1.54 5.16 5.16 0 011.78-.91 7.72 7.72 0 012.22-.31 6.82 6.82 0 012.07.3 4.85 4.85 0 011.68.91c.48.4.85.9 1.12 1.51.27.6.4 1.31.4 2.11 0 .51-.05.98-.16 1.4a4.68 4.68 0 01-.45 1.13c-.19.34-.42.65-.68.94-.26.29-.54.57-.85.84l-4.95 4.2h7.23v2.99H51.87v-3.6z"/><path fill="#2aaeff" d="M67.5 1.42a42.97 42.97 0 00-61.1 0 42.89 42.89 0 000 61.06c16.83 17 44.27 17 61.1 0a42.89 42.89 0 000-61.06"/><path fill="#558bec" d="M66.8 56.8a37.67 37.67 0 01-52.59 0 37.6 37.6 0 010-52.54 37.67 37.67 0 0152.58 0 37.6 37.6 0 010 52.54"/><path fill="#7f68da" d="M66.08 51.12a30.16 30.16 0 01-43.34 0 30.1 30.1 0 010-43.31 30.16 30.16 0 0143.34 0 30.1 30.1 0 010 43.31"/><path fill="#aa46c7" d="M66.8 45.44a25.66 25.66 0 01-35.53 0 25.6 25.6 0 010-35.5 25.66 25.66 0 0135.52 0 25.6 25.6 0 010 35.5"/><path fill="#d423b5" d="M65.37 39.76a17.38 17.38 0 01-25.58 0 17.35 17.35 0 010-25.56 17.38 17.38 0 0125.58 0 17.35 17.35 0 010 25.56"/><path fill="#ff00a2" d="M65.37 34.79a12.83 12.83 0 01-17.76 0 12.8 12.8 0 010-17.75 12.83 12.83 0 0117.76 0 12.8 12.8 0 010 17.75"/></g></svg>
```

File: img/w/wp.pl.svg [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+     id="g9"
+     transform="translate(-1.8199111,-8.5668746e-5)">
+    <path
+       fill-rule="evenodd"
+       clip-rule="evenodd"
+       d="M 16.99,21.381 5.248,31.663 V 11.345 L 16.99,21.38 Z M 8.474,9.752 l 27.69,1.886 -12.57,11.261 z m 10.961,13.72 3.218,2.75 a 1.42,1.42 0 0 0 1.906,-0.04 l 3.127,-2.848 8.751,8.078 -28.065,1.912 11.063,-9.853 z M 38.595,13.4 V 29.417 L 29.931,21.29 Z M 39.984,8.891 3.59,5.991 a 1.62,1.62 0 0 0 -1.77,1.63 v 27.826 a 1.62,1.62 0 0 0 1.631,1.636 c 0.046,0 0.093,-0.002 0.14,-0.006 l 36.393,-2.9 c 0.718,-0.056 1.28,-0.738 1.28,-1.55 V 10.442 c 0,-0.813 -0.562,-1.495 -1.28,-1.552 z"
+       fill="#ed1c24"
+       id="path7" />
+  </g>
+</svg>
```
Comment (Carlgo11) [inline img/w/wp.pl.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 94.85 43"><path fill="#ed1c24" d="M23.2 1.31a4.3 4.3 0 00-1.76 2.3c-.4 1.56.15 2.95.56 4.33L1.76 6A1.62 1.62 0 00-.01 7.62v27.83c0 .9.73 1.64 1.63 1.63 9.56-.74 19.12-1.51 28.68-2.27.63 2.32 1.78 4.6 3.52 5.99 7.31 4.81 19.18.72 19.81-7.98.86-7.73 1.28-13.85 2.09-20.85.04-.3.19-.45.44-.48.27-.02.45.09.55.38 1.76 7.58 3.05 13.75 4.7 20.61.22.96.61 1.88 1.16 2.7 3.83 3.5 8.51 3.57 11.95.38a8.39 8.39 0 002.35-4.43c.9-7.3 1.4-12.85 2.09-19.24 2.56.07 5.2.04 7.48.15 3.52.27 3 7.77.35 8.22-1.41.2-2.91.25-4.22.33-.6.04-1.08.3-1.45.77a4.66 4.66 0 00-.86 4.12c.12.51.36.99.71 1.38.35.43.9.64 1.45.57 2.06-.25 4.22-.5 6.06-.9a8.2 8.2 0 005.48-5.13c.82-2.64 1.12-5.35.74-7.89-.56-4.19-4.05-7.98-7.6-8.67-3.79-.47-7.96-.69-11.38-1a3.56 3.56 0 00-3.76 2.28c-1 6.2-1.45 13.03-1.9 18.72l-.33 4.48c-.02.27-.04.53-.08.8-.03.22-.14.34-.3.36a.47.47 0 01-.53-.35l-.13-.64c-1.17-5.98-2.36-11.96-3.55-17.94a14.58 14.58 0 00-1.2-3.7 8.98 8.98 0 00-4.02-4.13 11.24 11.24 0 00-8.15-.8 7 7 0 00-3.7 2.4 8.74 8.74 0 00-1.53 2.9c-1.3 9.16-1.92 17.32-2.62 25.08-.04.34-.18.48-.5.53-.38.06-.64-.1-.74-.5-2.03-9.44-4.77-19.03-6.92-27.52C34.75 2.59 31.88.61 28.88.1a8.25 8.25 0 00-5.68 1.21zM6.65 9.75l25.33 1.73.48 1.85-10.7 9.57zm-3.22 1.6l11.74 10.03L3.43 31.66zm29.82 5l3.37 12.92-8.5-7.98c2.17-2.17 3.82-3.77 5.13-4.93zm-7.38 6.98l8.75 8.08-28.07 1.91 11.06-9.85 3.22 2.75c.55.48 1.37.46 1.9-.04z"/></svg>
```

File: entries/o/o2.pl.json [hunk 8]
```diff
@@ -0,0 +1,14 @@
+{
+  "o2": {
+    "domain": "o2.pl",
+    "url": "https://www.o2.pl/",
```
Comment (Carlgo11) [inline entries/o/o2.pl.json]:
```suggestion
```

File: entries/o/o2.pl.json [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+    "tfa": [
+      "totp",
+      "custom-software"
+    ],
+    "documentation": "https://pomoc.o2.pl/1login/jak-wlaczyc-i-wylaczyc-weryfikacje-dwuetapowa",
```
Comment (Carlgo11) [inline entries/o/o2.pl.json:10]: Are o2 and WP the same service?
I see they use the same documentation page.

File: entries/o/o2.pl.json [hunk 10]
```diff
@@ -0,0 +1,14 @@
...
+    "keywords": [
+      "email"
+    ]
```
Comment (Carlgo11) [inline entries/o/o2.pl.json:16]:
```suggestion
    ],
    "regions": [
      "pl"
    ]
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/w/wp.pl.json (hunk 1 above)
Comment (mimi89999) [reply entries/w/wp.pl.json:5]: It's https://play.google.com/store/apps/details?id=pl.wp.onelogin
It supports TOTP offline mode and online mode where the user confirms login requests in the app.

File: entries/o/o2.pl.json (hunk 9 above)
Comment (mimi89999) [reply entries/o/o2.pl.json:10]: Both have the same owner, but they are different services.

**Comment by mimi89999** [comment]
Do I need to do further changes to this PR?

File: entries/w/wp.pl.json (hunk 1 above)
Comment (ApCoder123) [reply entries/w/wp.pl.json:5]: Please could you specify this with a `custom-software` field, like below:
```JSON
"custom-software": [
  "1login od WP"
]
```
There is more information on this in the [Other Properties](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#other-properties) section of the contributing guidelines.

**Comment by ApCoder123** [comment]
> Do I need to do further changes to this PR?

@mimi89999 Please could you confirm whether the TOTP only works with the "1login od WP" app or others such as Google Authenticator, Authy, etc... as mentioned by Carlgo11 in https://github.com/2factorauth/twofactorauth/pull/5979#discussion_r672729001

File: entries/w/wp.pl.json (hunk 2 above)
Comment (mimi89999) [reply entries/w/wp.pl.json]: No, it doesn't. It's TOTP, but they changed the URI scheme to `olotpauth`.

**Comment by mimi89999** [comment]
> > Do I need to do further changes to this PR?
> 
> @mimi89999 Please could you confirm whether the TOTP only works with the "1login od WP" app or others such as Google Authenticator, Authy, etc... as mentioned by Carlgo11 in [#5979 (comment)](https://github.com/2factorauth/twofactorauth/pull/5979#discussion_r672729001)

It doesn't because they changed the URI scheme to `olotpauth`. I removed `totp` from the list.

File: entries/w/wp.pl.json [hunk 11]
```diff
@@ -0,0 +1,18 @@
+{
+  "Wirtualna Polska ": {
```
Comment (ApCoder123) [inline entries/w/wp.pl.json]:
```suggestion
  "Wirtualna Polska": {
```

**Review by ApCoder123 — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**